benchmark-service/benchmark/benchmark.py

```diff
@@ -45,10 +45,12 @@ def get_benchmarks(app, file_path):
         stats['failed'] = d['failed']
         if not d['failed']:
             prod_of_means *= d['mean']
-            stats['f-stat'] = round(d['f-stat'], 6)
+            if d.get('f-stat') is not None:
```

Comment:
Let's write a wrapper function for all of these values in utils.py in case more of them are not defined. Something like this:

```python
def round_if_defined(x):
    if x is not None:
        return round(x, 6)
    return None
```

```python
stats['p-value'] = round_if_defined(d.get('p-value'))
```